composer.json

```diff
@@ -31,8 +31,8 @@
         }
     },
     "require-dev": {
-        "sabre/cs": "^1.0",
-        "phpunit/phpunit" : "*"
+        "friendsofphp/php-cs-fixer": "^2",
```

Comment:
As mentioned in the other PR, we should add the minor for the cs-fixer

Reply:
Pinned it to 2.16.*